rofl-containers/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rofl-containers"
-version = "0.3.5"
+version = "0.4.0"
```

Comment:
Are we bumping this on every PR?

Reply:
He he, this was started before the previous bump and I thought it would take longer. In the end it makes sense to just do 0.4.0.